cassandra_check/README.md

```diff
@@ -0,0 +1,29 @@
+# Cassandra Check
```

Comment:
We should change the name of this check, I think having "Cassandra" and "Cassandra Check" would be confusing.
At this point why not having a "Cassandra Nodetool" check? This would be explicit in what it does, plus we could support more `nodetool` commands other than `status` in the future with the same check.